src/main.ts

```diff
@@ -1,4 +1,4 @@
-export { default, rebuildNativeModules } from './rebuild';
+export { rebuild, rebuildNativeModules } from './rebuild';
```

Comment:
@paulcbetts @MarshallOfSound this is a breaking change -- shouldn't it have been released as a major version bump? It broke `electron-forge` -- I think [this](https://github.com/electron-userland/electron-forge/blob/master/src/util/rebuild.js#L1) line needs to be `import { rebuild } from 'electron-rebuild'`.